### PR TITLE
lcase the directory

### PR DESCRIPTION
**Pull request by rmarchandTen24**
This change is [Reviewable](https://reviewable.io/reviews/ten24/slatwall/5998)

### CODE REVIEW

File: org/Hibachi/ckfinder/config.cfm [hunk 1]
```diff
@@ -182,15 +182,15 @@ config.resourceType = arrayNew(1);
 config.resourceType[1] = structNew();
 config.resourceType[1].name = 'Files';
 config.resourceType[1].url = config.baseUrl & 'Files';
```
Comment (sumitverma) [inline org/Hibachi/ckfinder/config.cfm]: You will need to change URL as well

Comment (sumitverma) [reply org/Hibachi/ckfinder/config.cfm]: Also, I think this will be a braking change on case sensitive system, we will need to do it through config.

Comment (rmarchandTen24) [reply org/Hibachi/ckfinder/config.cfm]: @sumitverma so are you going to do it via the custom config overrider?

Comment (sumitverma) [reply org/Hibachi/ckfinder/config.cfm]: Right, I think we should make these file names variable that you can override from the custom config.

**Pull request closed without merging**